test/core/end2end/tests/filter_latency.cc

```diff
@@ -122,7 +122,7 @@ static void test_request(grpc_end2end_test_config config) {
   g_client_latency = gpr_time_0(GPR_TIMESPAN);
   g_server_latency = gpr_time_0(GPR_TIMESPAN);
   gpr_mu_unlock(&g_mu);
-  const gpr_timespec start_time = gpr_now(GPR_CLOCK_MONOTONIC);
+  const gpr_timespec start_time = gpr_now(GPR_CLOCK_REALTIME);
```

Comment:
@AspirinSJL I changed this test from `CLOCK_MONOTONIC` TO `CLOCK_REALTIME` because on windows `CLOCK_MONOTONIC` is always 0. Could you PTAL? thanks!